catalog/ui/catalog-ui-search/src/main/webapp/js/view/ingest/IngestModal.view.js

```diff
@@ -117,14 +117,17 @@ define([
                         type = 'JAR';
                     }
                 }
+                var parsedProgress = parseInt(data.loaded / data.total * 100, 10);
```

Comment:
Changed to use `|| 0`.  This `NaN` is caused by `data.loaded` not initially being set in the file data received from jQuery-File-Upload (https://github.com/blueimp/jQuery-File-Upload)